libs/remix-ui/panel/src/lib/plugins/panel-header.tsx

```diff
@@ -27,7 +28,10 @@ const RemixUIPanelHeader = (props: RemixPanelProps) => {
   return (
     <header className='d-flex flex-column'>
       <div className="swapitHeader px-3 pt-2 pb-0 d-flex flex-row">
-        <h6 className="mb-3" data-id='sidePanelSwapitTitle'>{plugin?.profile.displayName || plugin?.profile.name}</h6>
+        <h6 className="mb-3" data-id='sidePanelSwapitTitle'>
+          {/* @ts-ignore */}
+          <FormattedMessage id={plugin?.profile.intlId || 'defaultId'} defaultMessage={plugin?.profile.displayName || plugin?.profile.name} />
```

Comment:
is this possible to compute this id here instead of having this new propery `intlId` . like with `plugin?.profile.name + '.displayName'

Reply:
Yeah, that's a good idea. I just pushed another commit to update it.